compiler/src/iree/compiler/Dialect/Stream/Transforms/DumpStatistics.cpp

```diff
@@ -293,9 +293,9 @@ static void prettyPrintStreamInfo(const UsageInfo &usageInfo,
                                   llvm::raw_fd_ostream &os) {
   auto parentOp = executeOp->getParentOfType<mlir::FunctionOpInterface>();
 
-  prettyPrintItemHeader(
-      llvm::formatv("stream.cmd.execute", parentOp->getName().getStringRef()),
-      os);
+  prettyPrintItemHeader(llvm::formatv("stream.cmd.execute {0}",
+                                      parentOp->getName().getStringRef()),
```

Comment:
Did the original code add a space after 'execute'?

I still think this just concatenates two strings, I think this could be:
`(Twine("stream.cmd.execute ") +parentOp->getName().getStringRef()).str()`.

But fine to leave as-is if you don't want to change this.

Reply:
The original code had a bug `parentOp->getName().getStringRef()).str()` would never get printed in it but I think the intention was to print it. I can make the fixes you suggested.